easybuild/toolchains/linalg/intelmkl.py

```diff
@@ -84,6 +85,7 @@ def _set_blas_variables(self):
         interfacemap = {
             TC_CONSTANT_INTELCOMP: 'intel',
             TC_CONSTANT_GCC: 'gf',
+            TC_CONSTANT_PGI: 'intel',
```

Comment:
But its not, didn't I give a reference to the pgi linker page?

On 2 May 2016 9:00 p.m., "Kenneth Hoste" <notifications@github.com<mailto:notifications@github.com>> wrote:

In easybuild/toolchains/linalg/intelmkl.py<https://github.com/hpcugent/easybuild-framework/pull/1664#discussion_r61776195>:

> @@ -84,6 +85,7 @@ def _set_blas_variables(self):
>          interfacemap = {
>              TC_CONSTANT_INTELCOMP: 'intel',
>              TC_CONSTANT_GCC: 'gf',
> +            TC_CONSTANT_PGI: 'intel',


@ocaisa<https://github.com/ocaisa> this looks wrong?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/easybuild-framework/pull/1664/files/f88303f378701303559b06724b546944f8bf5a3f#r61776195>


------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
Forschungszentrum Juelich GmbH
52425 Juelich
Sitz der Gesellschaft: Juelich
Eingetragen im Handelsregister des Amtsgerichts Dueren Nr. HR B 3498
Vorsitzender des Aufsichtsrats: MinDir Dr. Karl Eugen Huthmacher
Geschaeftsfuehrung: Prof. Dr.-Ing. Wolfgang Marquardt (Vorsitzender),
Karsten Beneke (stellv. Vorsitzender), Prof. Dr.-Ing. Harald Bolt,
Prof. Dr. Sebastian M. Schmidt
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------